config/application.php

```diff
@@ -72,7 +72,6 @@
  * Custom Settings
  */
 define('AUTOMATIC_UPDATER_DISABLED', true);
-define('DISABLE_WP_CRON', env('DISABLE_WP_CRON') ?: false);
```

Comment:
what is going on here?

Reply:
This is not completely removed, but the define is moved into the environment configuration files. This is because this constant cannot be defined in the testing environment, as it will define it itself without checking if it has already been defined or not. 

The alternative to moving it to the configuration files (and not to `testing.php`) is to wrap this with a conditional to only define it if `WP_ENV !== 'testing'` which is a little less clean IMO.